packages/base/Page/src/TopBarItem/TopBarItem.tsx

```diff
@@ -27,20 +24,39 @@ export interface Props extends BaseProps, TextLabelProps {
   onMouseEnter?: (event: React.MouseEvent<HTMLElement, MouseEvent>) => void
 }
 
-const useStyles = makeStyles<Theme>(styles, { name: 'PicassoTopBarMenuItem' })
+const rootClasses = twJoin(
+  'min-[1280px]:text-gray-600 min-[1280px]:p-0 min-[1280px]:h-auto',
+  'min-[1280px]:w-auto min-[1280px]:m-0 min-[1280px]:flex-auto',
+  'min-[1280px]:[&_p]:text-sm'
+)
```

Comment:
```suggestion
const rootClasses = [
  'min-[1280px]:text-gray-600 min-[1280px]:p-0 min-[1280px]:h-auto',
  'min-[1280px]:w-auto min-[1280px]:m-0 min-[1280px]:flex-auto',
  'min-[1280px]:[&_p]:text-sm'
]
```

Is it nessesary to use `twJoin` here (and 3 other cases below)? Perhaps it could just be an array we pass to `twMerge` later. From what I understand `twJoin` is just a superset of `classnames` for resolving conditionals, which we don't have in this case.

Reply:
✅